.gitignore

```diff
@@ -17,6 +17,9 @@ yarn.lock
 /composer.lock
 
 .cache
+!/.cache/
```

Comment:
This ignores all `.gitkeep` files but `/.caches/.gitkeep`.
The `/.caches/` folder is needed for the `PHP Coding Standards` CI job.